app/views/admin/oral_history_access_requests/index.html.erb

```diff
@@ -55,7 +72,7 @@
           <td class="col-sm-4">
              <%= oh_request.intended_use %>
           </td>
-          <td class="col-sm-1">
+          <td class="col-sm-1 request-list-delivery-status">
```

Comment:
You added a class name even though you didn't add any styles targeting it? Doesn't hurt though!

Reply:
Yeah. I use it in the tests and it looks harmless enough.